tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/VesselTO.java

```diff
@@ -7,17 +7,20 @@
 import java.util.UUID;
 
 public record VesselTO(
-  UUID id,
   String vesselIMONumber,
   String name,
   String flag,
-  String callSignNumber,
+  String callSign,
+  String operatorCarrierCode, // <-
+  String operatorCarrierCodeListProvider // <-
+  /*
   CarrierTO vesselOperatorCarrier,
   Boolean isDummy,
   Float length,
   Float width,
   VesselType type,
   DimensionUnit dimensionUnit
+   */
```

Comment:
Left-over code or a TODO?

Reply:
Leftover, forgot to properly map vessels. Fixed now